src/main/java/it/gov/pagopa/payhub/activities/connector/sendnotification/client/SendClient.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.payhub.activities.connector.sendnotification.client;
+
+import it.gov.pagopa.payhub.activities.connector.sendnotification.config.SendApisHolder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
always use @Lazy on business @Service in the repo activites
```suggestion
@Lazy
@Service
```